cmd/pyroscope/command/adhoc.go

```diff
@@ -0,0 +1,56 @@
+package command
+
+import (
+	"github.com/spf13/cobra"
+
+	"github.com/pyroscope-io/pyroscope/pkg/adhoc"
+	"github.com/pyroscope-io/pyroscope/pkg/cli"
+	"github.com/pyroscope-io/pyroscope/pkg/config"
+)
+
+func newAdhocCmd(cfg *config.Adhoc) *cobra.Command {
+	vpr := newViper()
+
+	cmd := &cobra.Command{
+		Use:   "adhoc [flags] [<args>]",
+		Short: "Profile a process and save the results to be used in adhoc mode",
+		Long: `adhoc command is a complete toolset to profile a process and save the profiling
+results.
+
+These results are then available to be visualized through the 'Adhoc Profiling'
```

Comment:
I know we kinda dropped this idea, but now that I am thinking again, a `pyroscope adhoc report/analyze pyroscope.data` which opens the browser in the adhoc only mode would be kinda useful.